### ISSUE TITLE
Diaphragmatic hernia

### ISSUE THREAD

**Issue by drseb** [obophenotype/human-phenotype-ontology#4937]
**Preferred term label:**

LEFT DIAPHRAGMATIC HERNIA
RIGHT DIAPHRAGMATIC HERNIA

**Synonyms**

left-sided congenital diaphragmatic hernia
right-sided congenital diaphragmatic hernia

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

I will take care of this, if the classes are created

**Your nano-attribution (ORCID)**

_s_ to the _e_ to the _b_

**Comment by pnrobinson** [comment]
These are valid terms but might overlap with e.g. https://hpo.jax.org/app/browse/term/HP:0040048
I will try to explore this more closely.

**Comment by pnrobinson** [comment]
These are synonyms. If half of the diaphragm is absent, this will result in a diaphragmatic hernia in all cases, ,it is just a different way of saying things. Some of the terms are duplicated, e.g., Absent left hemidiaphragm and Aplasia of the left hemidiaphragm are synonymous. I will rearrange and merge.

**Comment by pnrobinson** [comment]
rearranged and added terms.
